3rdparty/bazel/org_openssl/BUILD.bazel

```diff
@@ -18,7 +18,7 @@ cc_test(
         ],
         "//conditions:default": [],
     }),
-    # TODO support OpenSSL on Mac
+    # TODO(elchead) support OpenSSL on Mac
```

Comment:
I think we are not really committed to do this. Should we remove the TODO or just keep for reference?

Reply:
I mean it's sort of your TODO, so you should know and you are the only one affected. If you don't need this, then I'll remove it.

Reply:
please remove it.